de/_posts/2019-12-07-rails-beginner-workshop.md

```diff
@@ -0,0 +1,55 @@
+---
+tags: beginner rails_beginner_workshop
+categories: de events
+slug: Anfänger_innen-Workshop
```

Comment:
Does it make sense to add the language here (looking at other events in the future)? And if the slug is supposed to become part of the URL, maybe change the ä into ae?

Reply:
when you say languages, do you mean like coding languages or spoken language? I considered it, but prefer not to tag/label stuff until it is explicitly used.

Reply:
I agree with switching `ä` to `ae`, because they are not interchangeable in the browser url unfortunately